benchmarks/markdown/package.json

```diff
@@ -8,8 +8,8 @@
     "serve": "gatsby serve"
   },
   "dependencies": {
-    "gatsby": "^2.0.9",
-    "gatsby-transformer-remark": "^2.1.4",
+    "gatsby": "latest",
```

Comment:
Instead of "latest", could you change these to the actual latest versions? I'd rather be explicit about versions.

Reply:
Done!